arangod/IResearch/GeoAnalyzer.cpp

```diff
@@ -62,6 +62,29 @@ constexpr auto kStr2Type =
         {"point", GeoJsonAnalyzerBase::Type::POINT},
     });
 
+constexpr auto kPoint2Str =
+    frozen::make_map<geo::coding::Options::Point, std::string_view>({
+        {geo::coding::Options::Point::S2Point, "s2Point"},
+        // TODO(MBkkt) Implement they
+        //  {geo::coding::Options::Point::S2PointShapeCompact, "..."},
+        //  {geo::coding::Options::Point::S2PointRegionCompact, "..."},
+        {geo::coding::Options::Point::S2LatLng, "latLngDouble"},
+        {geo::coding::Options::Point::S2LatLngInt, "latLngInt"},
+    });
+constexpr auto kStr2Point =
+    frozen::make_map<std::string_view, geo::coding::Options::Point>({
+        {"S2Point", geo::coding::Options::Point::S2Point},
```

Comment:
Is it intentional that here it is "S2Point" but line 67 says "s2Point" ?

Reply:
I mean shouldn't  `kPoint2Str` and `kStr2Point` be consistent?

Reply:
You re right

Reply:
Maybe use constants? as in line 279 we also use this values in return message

Reply:
`S2Point` looks better because `S2` is the library name.

Reply:
I decided if you're not insist to keep s2Point.
Because I want to keep all in camelCase